prody/proteins/mmtffile.py

```diff
@@ -322,7 +324,11 @@ def set_info(atomgroup, mmtf_data,get_bonds=False,altloc_sel='A'):
     mask = np.full(asize, True, dtype=bool)
     if altloc_sel != 'all':
         #mask out any unwanted alternative locations
-        mask = (altlocs == '') | (altlocs == altloc_sel)
+        default_altloc = altloc_sel if altloc_sel != None else 'A'
+        mask = (altlocs == '') | (altlocs == default_altloc)
+        if np.all(mask == False) and altloc_sel == None and len(altlocs):
+            #nothing selected, use first altloc; 6uwi
+            mask = altlocs == altlocs[0]            
```

Comment:
mask = (altlocs == '') | (altlocs == altlocs[0])

Reply:
If you get here, the mask was all false so you have no altlocs == ''

Reply:
Fair enough